src/main/java/com/ostapiuk/business/validator/PortalHealthCheckValidator.java

```diff
@@ -0,0 +1,20 @@
+package com.ostapiuk.business.validator;
+
+import com.ostapiuk.core.client.ReportPortalAPIClient;
+import com.ostapiuk.core.logger.Log;
+import com.ostapiuk.core.properties.DataProperties;
+import io.restassured.response.Response;
+import org.apache.http.HttpStatus;
+import org.testng.Assert;
+
+public class PortalHealthCheckValidator {
+
+    public void verifyReportPortalIsUp() {
+        Response portalResponse = new ReportPortalAPIClient().getReportPortalStatus();
```

Comment:
What does S from SOLID means?
if the method/class says that it should verify/validate something -> it should validate only and not make requests. I would suggest creating a Scenarios layer and keeping classes in appropriate places